UPGRADE-5.4.md

```diff
@@ -69,6 +69,13 @@ Messenger
  * Deprecate not setting the `delete_after_ack` config option (or DSN parameter) using the Redis transport,
    its default value will change to `true` in 6.0
 
+
+Notifier
+--------
+
+* The following data providers for `TransportTestCase` are now static: `toStringProvider()`, `supportedMessagesProvider()` and `unsupportedMessagesProvider()`
+* The `TransportTestCase::createTransport()` method is now static
```

Comment:
missing space at start of line (I think this is my most recurring review comment ever :) )
+ `[BC BREAK]` prefix

Reply:
Oops :sweat_smile: Thank you, updated as well!